test/inductor/test_aot_inductor.py

```diff
@@ -93,6 +93,11 @@ def run_compiled(cls, optimized, exported, example_inputs):
         flat_example_inputs = fx_pytree.tree_flatten_spec(
             (example_inputs, {}), exported.call_spec.in_spec
         )
+        # For scalar inputs, we need to wrap them into tensors
```

Comment:
@angelayi , I wonder if there is any existing util function in export that does this?

Reply:
Wait sorry, I'm confused why we need to wrap them as tensors. Does AOTInductor assume all inputs are tensors? I feel like that should be fixed then, because isn't there work to support symints as inputs?

Reply:
The AOTInductor generated code has a uniform entry function signature, which expects a vector of tensors. It's more flexible in the JIT mode where Python tuple can contain different types.  cc @ezyang

Reply:
Ultimately, AOTInductor should support int inputs, if only for efficiency reasons, because you are going to expect a nontrivial number of size-like arguments, for which it is pointless to wrap them into a full blown Tensor, just to immediately unwrap them again.

Reply:
cc @swolchok